server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -47,6 +47,13 @@ def parse(self, content, provider=None):
         events = []
 
         for item in items:
+            event = get_resource_service("events").find_one(
```

Comment:
I wouldn't do this onclusive specific, imo it's a generic feature

Reply:
I see, so in this case we have to do changes in core ?

Reply:
@petrjasek Maybe we could utilise the existing `disable_item_updates` config for ingests, and allow the specific feed parser to modify this behaviour?

Handling of this config in core:
https://github.com/superdesk/superdesk-core/blob/develop/superdesk/io/commands/update_ingest.py#L582
Config in UI:
![image](https://github.com/superdesk/superdesk-cp/assets/22425232/30569af6-4b34-4275-98ca-324694fe61eb)

Reply:
this configuration is used to disable the updates for all items, right? but here we have to prevent the updates from that item, which is edited in SD already or was unpublished. I wonder how we can utilize this because both circumstances are different, I guess.

Reply:
yep I was looking at that but we want to get updates only ignore it on items which were manually edited. eventually we might add switch for that (that's why I was suggesting to pass the provider to `should_update` method @devketanpro) but atm it's the only usecase we have